setup.examples/exim/mailwatch-exim.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Script to apply adjustments for exim
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+Webuser="$1"
+
+/etc/init.d/exim4 stop
```

Comment:
Not sure at 100% but more "independent" from type of init systems installed. 

- service may run scripts from either upstart, System V or systemd.
- select automatically between systemV, upstart or systemd
- service runs scripts in a predictable environment